### PR TITLE
config-tools: Add "ENFORCE_TURNOFF_AC" capability to xml

### PR DESCRIPTION
**Pull request by yuchuyang**
Add the capability to disable or enable #AC for Split-locked Access
through <scenario>.xmls. This was implemented through Kconfig. Move this
configuration from Kconfig to xmls.

Tracked-On: #5798
Signed-off-by: Yang,Yu-chu <yu-chu.yang@intel.com>

### CODE REVIEW

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br>No New Function Declaration/Definition Mismatch

File: misc/config_tools/schema/config.xsd [hunk 1]
```diff
@@ -104,6 +104,13 @@ multiboot2 protocol. If set to ``n``, GRUB's multiboot2 is not available as a
 boot option.</xs:documentation>
       </xs:annotation>
     </xs:element>
+    <xs:element name="ENFORCE_TURNOFF_AC" type="Boolean" default="n">
+      <xs:annotation>
+        <xs:documentation>Force to disable #AC for Split-locked Access.If CPU has #AC for
+split-locked access, HV enable it and VMs can't disable. Set this to enforce turn off that
+#AC, for community developer only.</xs:documentation>
```
Comment (junjiemao1) [inline misc/config_tools/schema/config.xsd:111]: `enable` -> `enables`
`enforce turn off` -> `enforce turning off`
`for community developer only` -> do you mean `for debugging purposes only`?

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br>No New Function Declaration/Definition Mismatch

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br>No New Function Declaration/Definition Mismatch

File: misc/config_tools/schema/config.xsd (hunk 1 above)
Comment (yuchuyang) [reply misc/config_tools/schema/config.xsd]: updated

**Comment by wenlingz** [comment]
@terryzouhao please help approve this PR, thanks

**Review by terryzouhao — APPROVED** [review]

**Pull request merged**